src/ConnectedMachine/examples/latest-2019-08-02/Get-AzConnectedMachine.md

```diff
@@ -0,0 +1,6 @@
+### Example 1:
+```powershell
+PS C:\> Get-AzConnectedMachine -SubscriptionId 67379433-5e19-4702-b39a-c0a03ca8d20c
+
```

Comment:
For cmdlets like a Get-*, there should be an output here.
Below is a sample.
```powershell
PS C:\> Get-AzBlockchainMember -Name dolauli001 -ResourceGroupName testgroup

Location Name       Type
-------- ----       ----
eastus   dolauli001 Microsoft.Blockchain/blockchainMembers
```

This command gets a blockchain member for a resource group.

Reply:
Added